packages/expo-media-library/src/MediaLibrary.ts

```diff
@@ -16,19 +16,28 @@ export type PermissionResponse = EXPermissionResponse & {
   /**
    * Indicates if your app has access to the whole or only part of the photo library. Possible values are:
    * - `'all'` if the user granted your app access to the whole photo library
-   * - `'limited'` if the user granted your app access only to selected photos (only available on Android API 34+ and iOS 14.0+)
+   * - `'limited'` if the user granted your app access only to selected photos (only available on Android API 14+ and iOS 14.0+)
    * - `'none'` if user denied or hasn't yet granted the permission
    */
   accessPrivileges?: 'all' | 'limited' | 'none';
 };
 
 /**
  * Determines the type of media that the app will ask the OS to get access to.
- * @platform android API 33+
+ * @platform android 13+
  */
 export type GranularPermission = 'audio' | 'photo' | 'video';
 
 export type MediaTypeValue = 'audio' | 'photo' | 'video' | 'unknown';
+
+/**
+ * On Android 14+, when calling [`presentPermissionsPickerAsync()`](#medialibrarypresentpermissionspickerasyncmediatypes),
+ * use this to specify the type(s) of media that the user will be granting access to.
```

Comment:
```suggestion
 * Represents the possible types of media that app will ask the OS to get access to when calling [`presentPermissionsPickerAsync()`](#medialibrarypresentpermissionspickerasyncmediatypes).
```

Let's avoid the platform repetition, focus on what the types represents, and move some parts out.